mautrix_twitter/portal.py

```diff
@@ -178,6 +179,7 @@ async def _upsert_reaction(
                 tw_receiver=self.receiver,
                 tw_sender=sender.twid,
                 reaction=reaction,
+                tw_reaction_id=reaction_id,
             ).insert()
```

Comment:
The existing/edit case also needs to update the reaction ID

Reply:
👍